packages/@aws-cdk/cdk/lib/cloudformation/resource.ts

```diff
@@ -48,6 +62,13 @@ export class Resource extends Referenceable {
    */
   public readonly resourceType: string;
 
+  /**
+   * AWS resource properties.
+   *
+   * This object is rendered via a call to "renderProperties(this.properties)".
+   */
+  protected readonly properties: any;
```

Comment:
Why do we need to expose this?

Reply:
https://github.com/awslabs/aws-cdk/blob/master/packages/@aws-cdk/cdk/lib/cloudformation/resource.ts#L68 I was reverting back to master. Isn't it `protected` there already?